libpod/oci_linux.go

```diff
@@ -246,7 +246,9 @@ func (r *OCIRuntime) createOCIContainer(ctr *Container, cgroupParent string, res
 	}
 
 	logDriver := KubernetesLogging
-	if ctr.LogDriver() != "" {
+	if ctr.LogDriver() == JSONLogging {
+		logrus.Debugf("json-file logging specified but not supported. Choosing k8s-file logging instead")
```

Comment:
Honestly I'd up this to an `Errorf()` - we want people to move over to using the supported driver so there are no surprises when we eventually implement the JSON log backend.

Reply:
fixed!